### DISCUSSION TITLE
DefectDojo can't import a report w/vulnerabilities from Nmap (but can import a report w/o vulnerabilities)

### DISCUSSION THREAD

**Discussion by superpalych** [DefectDojo/django-DefectDojo#10160] [Q&A]
I executed the command sudo nmap -O -sV 172.16.30.0/24 -oA 172-16-30-0
Nmap 7.94SVN created a report with hosts and open ports, but **without** vulnerabilities.
DefectDojo 2.12.0 **successfully** imported the report and displayed a list of hosts and open ports, but **without** vulnerabilities.
I executed the command sudo nmap -O -sV –script **vulners** 172.16.30.0/24 -oA 172-16-30-0 and command sudo nmap -sV –script **vulners** 172.16.30.0/24 -oA 172-16-30-0 (w/o -O)
Nmap created two reports with hosts, open ports **and** vulnerabilities.
DefectDojo was **unable** to import this reports with hosts, open ports and vulnerabilities. Error text: _An exception error occurred during the report import:Version of CPE not implemented_
Can the program import a vulnerability report? If so, how do I do it?

**Comment by kiblik** [comment]
Can you please open the issue and put there your report?
It looks like there is some bug in the parsing of the NMAP report and an example report would help to identify the exact issue.

**Reply by manuel-sommer** [reply]
Reminder, could you please share a report @superpalych ?

**Reply by resphantom** [reply]
Hi, I ran into a similar problem today with our POC, however I'm reluctant to provide the report because it does include company endpoints behind a firewall. 

What I can share is some of the output results of the ports and vulnerabilities:
![image](https://github.com/DefectDojo/django-DefectDojo/assets/78911523/3c1435ac-877a-4913-a122-e38f7ae7a513)

Example of problem:
![image](https://github.com/DefectDojo/django-DefectDojo/assets/78911523/498accd0-d607-4bf3-bb13-a091e47b7691)

What I do notice is that for some reason the nse 'vulners' script is what seems to be breaking and generating a broken vulnerability report. I don't think this is a Defectdojo issue. 

After locally running a vulnerable openssh container on port 2222, I do get a working vulnerability report using that same nse 'vulners' script, which does work on Defectdojo:
![image](https://github.com/DefectDojo/django-DefectDojo/assets/78911523/9840629c-3d04-45e7-96d7-7e1cc052596a)
![image](https://github.com/DefectDojo/django-DefectDojo/assets/78911523/45457070-248e-4e96-9b2d-335694667601)
